src/shared/components/progressIndicator/ProgressIndicator.tsx

```diff
@@ -129,13 +142,19 @@ export default class ProgressIndicator extends React.Component<
 
     render() {
         if (this.props.show) {
+            const detailedErrorMessages = getDetailedErrorMessages(this.items);
             return (
                 <div className={styles.container}>
                     <div className={styles['items-container']}>
                         {this.items.map(this.makeItem)}
                     </div>
                     {_.some(this.items, i => getItemStatus(i) === 'error') && (
-                        <ErrorMessage />
+                        <ErrorMessage
+                            message={detailedErrorMessages.join('\n')}
```

Comment:
Joining with `\n` is practically the same as joining with any white space. If we really need to format multiple lines of messages I think we need to pass a component instead of a simple string. Since encountering multiple detailed error messages is a low chance event, maybe it's okay to keep it like this for now?